File: TransCoder_tokenized_test_set_functions/java/NUMBER_DAYS_TANK_WILL_BECOME_EMPTY.java
static int minDaysToEmpty ( int C, int l ) {
  if ( l >= C ) return C;
  double eq_root = ( Math . sqrt ( 1 + 8 * ( C - l ) ) - 1 ) / 2;
  return ( int ) ( Math . ceil ( eq_root ) + l );
}

File: results/py/NUMBER_DAYS_TANK_WILL_BECOME_EMPTY.cpp
int f_filled ( int C , int l ) {
  if ( ( l >= C ) && ( l <= C ) ) return C ;
  double eqRoot = ( sqrt ( 1 + 8 * ( C - l ) ) - 1 ) / 2 ;
  return ceil ( eqRoot ) + l ;
}
